packages/arb-token-bridge-ui/src/util/deposits/helpers.ts

```diff
@@ -518,9 +525,8 @@ export async function fetchTeleporterDepositStatusData({
         l2L3Redeem && l2L3Redeem.status === ParentToChildMessageStatus.REDEEMED
           ? l2L3Redeem.childTxReceipt.transactionHash
           : undefined
-      const timestampResolved = await getTimestampResolved(
-        destinationChainProvider,
-        l3TxID
+      const timestampResolved = String(
+        await getTimestampResolved(destinationChainProvider, l3TxID)
```

Comment:
No need to `normalizeTimestamp(...)` here?

Reply:
`getTimestampResolved` returns a normalized timestamp